### PR TITLE
Reviewss

### PR DESCRIPTION
**Pull request by LuisDZurita**
hello.
we just add some changes on the footer and we add a new "ayuda" page. Juste let me know if is something is wrong.
thanks!

### CODE REVIEW

File: Ayuda.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+              </div>
+            </li>
+          </ul>
+        </div>
+        <div class="contact-elements">
+          <h3>E-mail</h3>
+          <ul>
+            <li><a href="mailto:wilber@gmail.com">wilber@gmail.com</a></li>
+            <li><a href="mailto:wilber@gmail.com">wilber@gmail.com</a></li>
+            <li><a href="mailto:wilber@gmail.com">wilber@gmail.com</a></li>
```
Comment (oreyes1991) [inline Ayuda.html]: Just curious, why this is repeated?  how come is not just one link?

File: cabanas.html [hunk 2]
```diff
@@ -1,76 +1,62 @@
...
-            </p>
-        </article>
+      <article class="post">
+        <h2>Quienes Somos?</h2>
+        <p>Pueblo pequeño</p>
+        <img src="img/cabanas-02_big.jpg" width="400px" />
+        <p>texto del articulo de prueba</p>
+      </article>
+      <article class="post">
+        <h2>el guevo!!</h2>
```
Comment (oreyes1991) [inline cabanas.html]: we should start removing this testing text, if there is still not clear what is going to be here we should replace it for a `Lorem Ipsum`

File: cabanas.html [hunk 3]
```diff
@@ -1,76 +1,62 @@
...
-        <p>
-            copyRai "BBCoffe" 2022
-        </p>
-    
+      <section>
+        <a href="#titulo"> Ir arriba </a>
+        <br />
+        <a href="mailto:luis@gmail.com"> contactanos!! </a>
+      </section>
+      <p>copyRai "BBCoffe" 2022</p>
```
Comment (oreyes1991) [inline cabanas.html]: can we get rid of this?

File: codejs.js [hunk 4]
```diff
@@ -1,91 +1,86 @@
-
-
-const cabanas = document.querySelectorAll (".cabana");
-const poput = document.querySelector('.back-Ground');
-const Xboton = document.querySelector('.Xboton');
+const cabanas = document.querySelectorAll(".cabana");
+const poput = document.querySelector(".back-Ground");
+const Xboton = document.querySelector(".Xboton");
```
Comment (oreyes1991) [inline codejs.js]: nit: const in JS should be camel case
```suggestion
const xButton = document.querySelector(".Xboton");
```

File: codejs.js [hunk 5]
```diff
@@ -1,91 +1,86 @@
...
-if (Xboton){
-    Xboton.addEventListener('click', () => {
-        poput.classList.add('hide');
-    });  
+for (let i = 0; i < cabanas.length; i++) {
+  const aCabana = cabanas[i];
+  aCabana.addEventListener("click", () => {
+    currentCabana = i;
+    poput.classList.remove("hide");
+  });
```
Comment (oreyes1991) [inline codejs.js]: nit: strings in JS should be single quote, not double.
```suggestion
  aCabana.addEventListener('click', () => {
    currentCabana = i;
    poput.classList.remove('hide');
  });
```

File: codejs.js [hunk 6]
```diff
@@ -1,91 +1,86 @@
...
+}
+if (Xboton) {
+  Xboton.addEventListener("click", () => {
+    poput.classList.add("hide");
+  });
```
Comment (oreyes1991) [inline codejs.js]: nit: same here
```suggestion
if (xButton) {
  xButton.addEventListener('click', () => {
    poput.classList.add('hide');
  });
```

File: codejs.js [hunk 7]
```diff
@@ -1,91 +1,86 @@
...
+  },
+  {
+    name: "cabana 2",
+    imgMosaico: "img/cabana2.jpg",
+    imgFullScren: "img/cabana2.jpg",
+  },
+  {
+    name: "cabana 3",
+    imgMosaico: "img/cabana2.jpg",
+    imgFullScren: "img/cabana2.jpg",
```
Comment (oreyes1991) [inline codejs.js]: nit: this should also be single quotes.

File: codejs.js [hunk 8]
```diff
@@ -95,86 +90,87 @@ const imgGallery = dataGallery.map((imgPP, i) =>{
...
-    }console.log(currentPhoto)
-    
-    eBoton.forEach((btn, i) =>{
-        btn.addEventListener('click',() =>{
-        const backGround= document.querySelector(`#a${i}`)
-        backGround.classList.add('hide');    
-        })
-    })
-};
+const mainGallery = document.querySelector(".gallery");
```
Comment (oreyes1991) [inline codejs.js]: same here

File: index.html [hunk 9]
```diff
@@ -1,132 +1,228 @@
...
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/ab4e6dc9fe.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
```
Comment (oreyes1991) [inline index.html:19]: Niceee 😄

**Review by oreyes1991 — CHANGES_REQUESTED** [review]
Good job man, I just left some small change requests

File: cabanas.html (hunk 2 above)
Comment (LuisDZurita) [reply cabanas.html]: I'm really sorry. we delete this page.

**Review by oreyes1991 — APPROVED** [review]

**Pull request merged**